### DISCUSSION TITLE
Will updating rpi os erase the OneInchEye installation

### DISCUSSION THREAD

**Discussion by Bebowl** [will127534/OneInchEye#23] [Q&A]
Hi @will127534 ,

Hope you're well. I have built several oneincheye modules with JLCPCB and it works like a charm. 
Thanks again for sharing this great work.

I have one question, I (of course) use you fork release of libcamera through python to talk with the IMX283 sensor, but I would like to know if I can safely update my Rpi OS with last patches without erasing your libcamera installation ?

Thanks for your help

**Comment by will127534** [comment]
I'll be much worried about the kernel drivers, the current method to install it is via DKMS, which is basically compile against current running kernel header and some symbol file, which if you update the kernel will have to be re-installed, and if the kernel version doesn't have a corresponding linux-header package ready, the DKMS installation will fail and you will have to download the kernel and compile yourself.

For the libcamera, I think you can go ahead with the RPI libcamera branch now, IMX283 kernel driver has been upstream already for Kernel v6.10 (Thanks to @kbingham and @uajain) so the libcamera support has also been merged to the latest RPI libcamera fork that you don't need my libcamera fork for IMX283 now.
